[2025-06-29T12:32:49.784+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_kpi_daily_batch.run_batch_youtube_etl manual__2025-06-29T12:32:49.050692+00:00 [queued]>
[2025-06-29T12:32:49.788+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_kpi_daily_batch.run_batch_youtube_etl manual__2025-06-29T12:32:49.050692+00:00 [queued]>
[2025-06-29T12:32:49.788+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-29T12:32:49.883+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_batch_youtube_etl> on 2025-06-29 12:32:49.050692+00:00
[2025-06-29T12:32:49.886+0000] {standard_task_runner.py:60} INFO - Started process 175 to run task
[2025-06-29T12:32:49.888+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'youtube_kpi_daily_batch', 'run_batch_youtube_etl', 'manual__2025-06-29T12:32:49.050692+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/youtube_etl_dag.py', '--cfg-path', '/tmp/tmpr3s4k0j1']
[2025-06-29T12:32:49.889+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask run_batch_youtube_etl
[2025-06-29T12:32:49.899+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-29T12:32:49.919+0000] {task_command.py:423} INFO - Running <TaskInstance: youtube_kpi_daily_batch.run_batch_youtube_etl manual__2025-06-29T12:32:49.050692+00:00 [running]> on host d0ff4814d978
[2025-06-29T12:32:49.953+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='yonatan' AIRFLOW_CTX_DAG_ID='youtube_kpi_daily_batch' AIRFLOW_CTX_TASK_ID='run_batch_youtube_etl' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T12:32:49.050692+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-29T12:32:49.050692+00:00'
[2025-06-29T12:32:49.967+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCvtRTOMP2TqYqu51xNrqAzg
[2025-06-29T12:32:50.710+0000] {logging_mixin.py:188} INFO - Failed for UCvtRTOMP2TqYqu51xNrqAzg: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:50.711+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UC0ZV6M2THA81QT9hrVWJG3A
[2025-06-29T12:32:51.612+0000] {logging_mixin.py:188} INFO - Failed for UC0ZV6M2THA81QT9hrVWJG3A: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:51.612+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCYzPXprvl5Y-Sf0g4vX-m6g
[2025-06-29T12:32:52.292+0000] {logging_mixin.py:188} INFO - Failed for UCYzPXprvl5Y-Sf0g4vX-m6g: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:52.293+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCbg_UMjlHJg_19SZckaKajg
[2025-06-29T12:32:53.076+0000] {logging_mixin.py:188} INFO - Failed for UCbg_UMjlHJg_19SZckaKajg: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:53.077+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCDVPcEbVLQgLZX0Rt6jo34A
[2025-06-29T12:32:53.718+0000] {logging_mixin.py:188} INFO - Failed for UCDVPcEbVLQgLZX0Rt6jo34A: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:53.718+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UC5zJwsFtEs9WYe3A76p7xIA
[2025-06-29T12:32:54.706+0000] {logging_mixin.py:188} INFO - Failed for UC5zJwsFtEs9WYe3A76p7xIA: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:54.706+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCDSJCBYqL7VQrlXfhr1RtwA
[2025-06-29T12:32:55.390+0000] {logging_mixin.py:188} INFO - Failed for UCDSJCBYqL7VQrlXfhr1RtwA: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:55.390+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UC2C_jShtL725hvbm1arSV9w
[2025-06-29T12:32:56.163+0000] {logging_mixin.py:188} INFO - Failed for UC2C_jShtL725hvbm1arSV9w: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:56.164+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UCZDdF_p-L88NWVpzF0vjvMQ
[2025-06-29T12:32:56.978+0000] {logging_mixin.py:188} INFO - Failed for UCZDdF_p-L88NWVpzF0vjvMQ: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:56.978+0000] {logging_mixin.py:188} INFO - Running ETL for channel: UC648rgJOboZlgcDbW00vTSA
[2025-06-29T12:32:57.804+0000] {logging_mixin.py:188} INFO - Failed for UC648rgJOboZlgcDbW00vTSA: column "daily_view_delta" of relation "youtube_channel_stats" does not exist
LINE 4:             daily_view_delta, daily_subscriber_delta, daily_...
                    ^
[2025-06-29T12:32:57.805+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-29T12:32:57.815+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=youtube_kpi_daily_batch, task_id=run_batch_youtube_etl, execution_date=20250629T123249, start_date=20250629T123249, end_date=20250629T123257
[2025-06-29T12:32:57.843+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-29T12:32:57.855+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
